components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -170,9 +172,7 @@ interface PageHandler {
   // message. |turn_id| is the index of the message in the
   // current conversation.
   RateMessage(bool is_liked, uint32 turn_id) => (string? rating_id);
-  SendFeedback(string category,
-      string feedback,
-          string rating_id) => (bool is_success);
+  SendFeedback(string category, string feedback, string rating_id, bool send_page_url) => (bool is_success);
```

Comment:
nit: just noting that half this PR calls this `hostname` (e.g. `send_hostname` or `SiteInfo::hostname`, whereas the other half calls this `url` (e.g. `send_page_url`). That's a little confusing and IMO we should keep it consistent for the same piece of data.

Reply:
changed